src/object/C4ObjectInfoList.cpp

```diff
@@ -39,95 +39,122 @@ C4ObjectInfoList::~C4ObjectInfoList()
 
 void C4ObjectInfoList::Default()
 {
-	First=nullptr;
-	iNumCreated=0;
+	First = nullptr;
+	iNumCreated = 0;
 }
 
 void C4ObjectInfoList::Clear()
 {
 	C4ObjectInfo *next;
 	while (First)
 	{
-		next=First->Next;
+		next = First->Next;
 		delete First;
-		First=next;
+		First = next;
 	}
 }
 
 int32_t C4ObjectInfoList::Load(C4Group &hGroup)
 {
 	C4ObjectInfo *ninf;
-	int32_t infn=0;
-	char entryname[256+1];
+	int32_t infn = 0;
+	char entryname[256 + 1];
 
 	// Search all oci files
 	hGroup.ResetSearch();
-	while (hGroup.FindNextEntry(C4CFN_ObjectInfoFiles,entryname))
-		if ((ninf=new C4ObjectInfo))
+	while (hGroup.FindNextEntry(C4CFN_ObjectInfoFiles, entryname))
+	{
+		if ((ninf = new C4ObjectInfo))
 		{
-			if (ninf->Load(hGroup,entryname))  { Add(ninf); infn++; }
-			else delete ninf;
+			if (ninf->Load(hGroup,entryname))
+			{
+				Add(ninf);
+				infn++;
+			}
+			else
+			{
+				delete ninf;
+			}
 		}
+	}
 
 	// Search subfolders
 	hGroup.ResetSearch();
 	while (hGroup.FindNextEntry("*", entryname))
 	{
 		C4Group ItemGroup;
 		if (ItemGroup.OpenAsChild(&hGroup, entryname))
+		{
 			Load(ItemGroup);
+		}
 	}
 
 	return infn;
 }
 
 bool C4ObjectInfoList::Add(C4ObjectInfo *pInfo)
 {
-	if (!pInfo) return false;
-	pInfo->Next=First;
-	First=pInfo;
+	if (!pInfo)
+	{
+		return false;
+	}
+	pInfo->Next = First;
+	First = pInfo;
 	return true;
 }
 
 void C4ObjectInfoList::MakeValidName(char *sName)
 {
 	char tstr[_MAX_PATH];
```

Comment:
`_MAX_PATH` doesn't make sense at all here.

Reply:
what should we do with it?

Reply:
To be honest, I do not understand how the function makes a valid name.